net/quagga/src/opnsense/mvc/app/models/OPNsense/Quagga/OSPF6.php

```diff
@@ -0,0 +1,31 @@
+<?php
+namespace OPNsense\Quagga;
+
+use OPNsense\Base\BaseModel;
+
+
+/*
```

Comment:
copyright should always be the first block in the file. this leaked into quagga because one file started this way ;)